Database/Corrections/Automatic/sodBaseQuests.lua

```diff
@@ -493,15 +493,48 @@ function SeasonOfDiscovery:LoadBaseQuests()
             [questKeys.objectivesText] = {"Travel to the Echo Isles and use the offering at Bwonsamdi's shrine."},
             [questKeys.objectives] = nil,
         },
+        [78226] = { -- This quest is not in the game. It's a placeholder fake quest to make the quest chain work.
+            [questKeys.name] = "Trial of The Owls",
+            [questKeys.startedBy] = nil,
+            [questKeys.finishedBy] = {{12042,}},
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 25,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.DRUID,
+            [questKeys.objectivesText] = {"Obtain 1 symbol of the owl to start the Trial of The Owls quest chain."},
+            [questKeys.objectives] = {nil,nil,{{210026}}},
+        },
+        [78227] = { -- This quest is not in the game. It's a placeholder fake quest to make the quest chain work.
+            [questKeys.name] = "Trial of The Owls",
+            [questKeys.startedBy] = nil,
+            [questKeys.finishedBy] = {{12042,}},
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 25,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.DRUID,
+            [questKeys.objectivesText] = {"Obtain 1 symbol of the owl to start the Trial of The Owls quest chain."},
+            [questKeys.objectives] = {nil,nil,{{210043}}},
+        },
+        [78228] = { -- This quest is not in the game. It's a placeholder fake quest to make the quest chain work.
+            [questKeys.name] = "Trial of The Owls",
+            [questKeys.startedBy] = nil,
+            [questKeys.finishedBy] = {{12042,}},
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 25,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.DRUID,
+            [questKeys.objectivesText] = {"Obtain 1 symbol of the owl to start the Trial of The Owls quest chain."},
+            [questKeys.objectives] = {nil,nil,{{210044}}},
+        },
```

Comment:
First of all: No manual edits in the base file!

Moreover you should not use these IDs, because they are in the "valid quest ID" range and Blizzard might decide to use them for actual quests in the future. Instead you should move these entirely into the questFixes file. At the bottom of the corrections there you'll fine a "fake quests" section, starting with ID 90000.